### PR TITLE
Add mochawesome test reporter to the project.

### PR DESCRIPTION
**Pull request by bartzet**
# Problem
Our UI test reports were just the default terminal reports.

# Solution
Add mochawesome test reported to the project so when the failures in the UI tests occur, it will be easier to investigate.

## Checklist
_put `n/a` if item is not relevant to PR changes_

- [ ] Have the CHANGELOG, README, MIGRATION, RELEASE_GUIDELINES docs been updated?
- [ ] Have new automated tests been implemented?
- [ ] Have new manual tests been written down?
- [ ] Have any new strings been translated?

### CODE REVIEW

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-866-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-866-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://866-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-866-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-866-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://866-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-866-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-866-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://866-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-866-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-866-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://866-pr-onfido-sdk-ui-onfido.surge.sh

File: test/main.js [hunk 1]
```diff
@@ -95,6 +95,14 @@ const createBrowser = async (browser, testCase) => {
 const createMocha = (driver, testCase) => {
   // Create our Mocha instance
   const mocha = new Mocha({
+    reporter: 'mochawesome',
+    reporterOptions: {
+      overwrite: true,
```
Comment (stefaniacardenas) [inline test/main.js]: Does this mean that every time we run it the old report will be overridden?

File: test/main.js [hunk 2]
```diff
@@ -95,6 +95,14 @@ const createBrowser = async (browser, testCase) => {
...
+      reportTitle: 'UI Tests',
+      reportFilename: 'UITestReport',
+      reportDir: './dist/reports/UITestsReport',
+      assetsDir: './dist/reports/UITestsReport',
```
Comment (stefaniacardenas) [inline test/main.js]: What's this for?

File: test/main.js (hunk 1 above)
Comment (bartzet) [reply test/main.js]: Yes.

File: test/main.js (hunk 2 above)
Comment (bartzet) [reply test/main.js]: That's Path to save report assets - js/css files.
 
`reportDir` is only for the json and html files generated.

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-866-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-866-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://866-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-866-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-866-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://866-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-866-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-866-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://866-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-866-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-866-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://866-pr-onfido-sdk-ui-onfido.surge.sh

**Pull request merged**